docs/developer_guide/Continous_Integration.md

```diff
@@ -0,0 +1,32 @@
+# The Continous Integration
```

Comment:
Every documentation file name only uses lower case characters. Can you change this file name?